dataactvalidator/app.py

```diff
@@ -17,6 +17,13 @@
 from dataactvalidator.validation_handlers.validationError import ValidationError
 from dataactvalidator.validation_handlers.validationManager import ValidationManager
 
+# Datadog import
+USE_DATADOG = False
```

Comment:
Add a comment here to let developers know that this value is being changed with Ansible, so nobody deletes it thinking the next `if` can never be reached.